cupy/linalg/decomposition.py

```diff
@@ -82,6 +85,12 @@ def qr(a, mode='reduced'):
             ``R`` with dimensions ``(M, K)``, ``(K, N)``, where
             ``K = min(M, N)``.
 
+    Returns:
+        cupy.ndarray, or tuple of :class:`~cupy.ndarray`:
```

Comment:
Link of the first ndarray is disrupted. How about "`cupy.ndarray or tuple of ndarray`" (without markup)?